docs/site/Customizing-Server-configuration.md

```diff
@@ -0,0 +1,281 @@
+---
```

Comment:
Suggest to rename the file name (just changing case) to be `Customizing-server-configuration.md` to be consistent with other file naming convention. In this case, we might need to change the permalink to be consistent as well. Thanks.

Reply:
Done